packages/widget-chat/src/actions/share.js

```diff
@@ -0,0 +1,28 @@
+export const FETCH_SHARE = `FETCH_SHARE`;
+export function fetchShare(fileObject) {
+  return {
+    type: FETCH_SHARE,
+    payload: {
+      fileObject
+    }
+  };
+}
+
+export const RECEIVE_SHARE = `RECEIVE_SHARE`;
+export function receiveShare(payload) {
```

Comment:
I like this way of receiving the `payload` object and mapping it to the key "payload". Good idea!